src/components/Form/DateControl/DateControl.jsx

```diff
@@ -269,10 +269,30 @@ export default class DateControl extends ValidationElement {
       let props = null
       if (date && !existingErr) {
         // Prepare some properties for the error testing
+        const buildValidators = (props) => {
```

Comment:
Since its going to require more work than we originally thought to handle validation against multiple dates, should we revert this section to what we currently have in the develop branch?